org/ystia/experimental/consul/linux/ansible/playbooks/consul_configure.yaml

```diff
@@ -53,10 +86,26 @@
           (WAN_ADDRESS | trim == '')
         )
 
+    - name:
+      set_fact:
+        subjectAltName: "{{ IP_ADDRESS }}"
+      when: consul_secured
+
+    - name:
+      set_fact:
+        subjectAltName: "{{ WAN_ADDRESS }}"
```

Comment:
In this case we could have both: the WAN address for the access from a remote datacenter, and the IP_ADDRESS for consul members within the same datacenter